src/components/app/details/triggerView/workflow/nodes/triggerCINode.tsx

```diff
@@ -58,7 +58,7 @@ export class TriggerCINode extends Component<TriggerCINodeProps> {
         else
             return (
                 <div className="dc__cd-trigger-status" style={{ color: TriggerStatus[status] }}>
-                    {this.props.status}
+                    {this.props.status.toLowerCase() === 'cancelled' ? 'ABORTED' : this.props.status}
```

Comment:
{this.props.status && this.props.status.toLowerCase() === 'cancelled' ? 'ABORTED' : this.props.status}
(null check basically)